### PR TITLE
added endpoint to create new school/admin

### PR DESCRIPTION
**Pull request by saxonewt**
# Conflicts:
#	main.go

### CODE REVIEW

File: main.go [hunk 1]
```diff
@@ -183,41 +182,56 @@ func createRouterClock(db *bolt.DB, clock Clock) *mux.Router {
...
 	}
 
 	_, err = CreateSchoolAdmin(db, admin)
 	if err != nil {
 		lgr.Printf("ERROR school admin is not created: %v", err)
-		return
+		return err
 	}
 
-	admin.Role = UserRoleTeacher
```
Comment (saxonewt) [inline main.go:210]: this is a bad idea to reuse an object to create a new record in db

File: main.go [hunk 2]
```diff
@@ -183,41 +182,59 @@ func createRouterClock(db *bolt.DB, clock Clock) *mux.Router {
...
-	admin.Name = "teacher@admin.com"
-	admin.Email = "teacher@admin.com"
+	tEmail := newSchoolRequest.Email[:1] + "." + newSchoolRequest.Email[1:]
```
Comment (saxonewt) [inline main.go:219]: teacher for admin!

if you have to create a new user - need to have a unique email for them

I add a dot after 1st letter in the email - it makes email different from original - but all email-servers handle it and route emails to the same postbox

Comment (anglofrog) [reply main.go:219]: not necessary since I will only make 1 teacher admin

File: main.go (hunk 1 above)
Comment (saxonewt) [reply main.go:210]: this comment is only about programming practice 

you used an object to create admin
then modified few fields to create another - this is the bad practice. you have to treat the objects as immutable and if you need to modified object you create new one

File: main.go [hunk 3]
```diff
@@ -183,41 +182,56 @@ func createRouterClock(db *bolt.DB, clock Clock) *mux.Router {
...
 		lgr.Printf("ERROR school admin is not created: %v", err)
-		return
+		return err
 	}
 
-	admin.Role = UserRoleTeacher
-	admin.Name = "teacher@admin.com"
-	admin.Email = "teacher@admin.com"
+	teacher := UserInfo{
+		Name:        "t_" + newSchoolRequest.Email,
```
[the remark this answers is not in the document]
Comment (saxonewt) [reply main.go]: you can continue modifying in `createNewSchool` function - I have changed only email for teacher, but I can't figure out how a teacher from one school can pay something in another

and it sounds strange to share a user among different admins for payments

is it bad to have individual teacher for every admin?

[the remark this answers is not in the document]
Comment (anglofrog) [reply main.go]: there will be one teacher account used to make pay/charge for all admins at one school

[the remark this answers is not in the document]
Comment (anglofrog) [reply main.go]: I could use different teacher for each admin but that will increase complexity and I am unsure that I want to do that

**Pull request merged**